### ISSUE TITLE
Can't use Docker on Windows. open //./pipe/docker_engine: The system cannot find the file specified

### ISSUE THREAD

**Issue by kamalpreetsingh24** [docker/for-win#5919]
I am using Docker Desktop 2.2.0.3 (42716) stable release in Windows 8.1 VM.
Currently, using Linux Containers and this is what I get If I run 'Docker Version' in Powershell.

PS C:\WINDOWS\system32> docker version
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea
 Built:             Wed Nov 13 07:22:37 2019
 OS/Arch:           windows/amd64
 Experimental:      false
Error response from daemon: open \\.\pipe\docker_engine_linux: The system cannot find the file specified.

However, If I restart Docker Desktop then it works fine but only for some time (say 10 to 15 minutes), 
PS C:\WINDOWS\system32> docker version
Client: Docker Engine - Community
 Version:           19.03.5
 API version:       1.40
 Go version:        go1.12.12
 Git commit:        633a0ea
 Built:             Wed Nov 13 07:22:37 2019
 OS/Arch:           windows/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          19.03.5
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.12.12
  Git commit:       633a0ea
  Built:            Wed Nov 13 07:29:19 2019
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          v1.2.10
  GitCommit:        b34a5c8af56e510852c35414db4c1f4fa6172339
 runc:
  Version:          1.0.0-rc8+dev
  GitCommit:        3e425f80a8c931f88e6d94a8c831b9d5aa481657
 docker-init:
  Version:          0.18.0
  GitCommit:        fec3683

"docker run hello-world" also confirms everything is looking good
"This message shows that your installation appears to be working correctly."

I am trying to build a Docker Image from the DockerFile and the base image which I am using is 
alpine.

I expect about 20 minutes of time to build the Docker image but because of Docker Desktop stopping frequently, the image build process halts.

Is there any workaround to fix this issue?

**Comment by afarid** [comment]
Fixed after restarting windows machine.

**Comment by binhex** [comment]
duplicate of https://github.com/docker/for-win/issues/4495

**Comment by opolancoh** [comment]
Restarting Docker or Windows solves problem, but it goes down again after 15 mins.

**Comment by Mechatronian** [comment]
same issue

**Comment by joseucarvajal** [comment]
> same issue

I have solved by doing this:

cd C:\Program Files\Docker\Docker\resources
dockerd.exe
Now execute 'docker version' on a new powershell as administrator

**Comment by 10152672** [comment]
> > same issue
> 
> I have solved by doing this:
> 
> cd C:\Program Files\Docker\Docker\resources
> dockerd.exe
> Now execute 'docker version' on a new powershell as administrator

I love you man!

**Comment by gyssels** [comment]
https://github.com/docker/for-win/issues/5919❯ .\dockerd.exe
time="2020-08-19T17:52:28.785589600+02:00" level=info msg="Starting up"
failed to load listeners: open //./pipe/docker_engine: Access is denied.

**Comment by kalyankalapala** [comment]
> dockerd.exe

Not working for me

**Comment by neopanda90** [comment]
> > same issue
> 
> I have solved by doing this:
> 
> cd C:\Program Files\Docker\Docker\resources
> dockerd.exe
> Now execute 'docker version' on a new powershell as administrator

Thank You

**Comment by Sean12697** [comment]
> > same issue
> 
> I have solved by doing this:
> 
> cd C:\Program Files\Docker\Docker\resources
> dockerd.exe
> Now execute 'docker version' on a new powershell as administrator

It worked for me, cheers! 😁

**Comment by Allen2316** [comment]
> > dockerd.exe
> 
> No funciona para mi

In other powerShell as administrator cd path/Dockerfile open file

**Comment by gizzm0** [comment]
Not working for me

**Comment by andras-varro** [comment]
I had the same issue when using a Hyper-V (Windows 10 as guest) as template for Packer. The issue was caused by insufficient system resources. By default Hyper-V gives 1Gb RAM and 1 virtual core. The solution was to specify in the Packer script to configure the Hyper-V to use 16Gb RAM and 2 virtual cores.
```
"builders": [
    {
    "type": "hyperv-vmcx",
    "memory": "16384",
    "cpus": "2",
[...]
   }
}
[...]
```
I hope this helps others, it took a few hours until I realized, that Hyper-V does not keep the VM settings on export.

**Comment by maxiride** [comment]
As pointed out by @binhex this is a duplicate of #4495, please concentrate all efforts on a single ticket.

**Comment by DSI-STEVET** [comment]
I could not get Docker Desktop to work.   I installed Docker, uninstalled Docker, tried the trouble shooting tools.  I changed files settings form memory in the Docker configuration files.   I tried running the program from the command line as described by Allen2316.   The program would always fail to start.    I installed Docker Desktop in the following sequence and it worked.    I would suggest going through these steps to confirm a clean install before spending a lot of time with configurations and alternate run methods on your computer.

1. Uninstall Docker Desktop if it is installed.
2. I restarted my computer.
2. I installed Docker Desktop for Windows after the fresh boot.   I did not run Docker Desktop as this caused problem before.
3. I restarted the computer and let Docker Desktop load at startup.
4. Docker Desktop loaded correctly and has not given problems since.
